README.md

```diff
@@ -1,12 +1,144 @@
-# Welcome to your CDK TypeScript Construct Library project
+# HTTP Lambda Interceptor
 
-You should explore the contents of this project. It demonstrates a CDK Construct Library that includes a construct (`LambdaHttpInterceptor`)
-which contains an Amazon SQS queue that is subscribed to an Amazon SNS topic.
+This library enables you to run integration tests on deployed lambda without worrying about calling external services by mocking them even though lambda are already deployed.
```

Comment:
This library enables you intercept and mock HTTP call made by an AWS Lambda. It's useful for integration testing.

Performing integration tests on deployed resources allows testing applications in the best prod alike environment. Thanks to the interceptor, lambda can be tested without interacting with costly external endpoints or non preprod external endpoints for example.